File: SuzumesDeepDungeon/Models/ExternalApi.cs
namespace SuzumesDeepDungeon.Models
{
    public class ExternalApi
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public string HashKey { get; set; }
        public User? User { get; set; }
        public bool IsActive { get; set; } = true;

        public int? UserId { get; set; }
        public DateTime Created { get; set; }
    }
}

File: SuzumesDeepDungeon/DeepDungeonClient/src/app/models/external-api.ts
import { User } from "./user";

export interface ExternalApi {
  id?: number;
  name?: string;
  description?: string;
  key?: string;
  userId?: number;
  user?: User;
  isActive?: boolean;
  created?: Date | string; // В зависимости от способа обработки дат
}